pl_examples/basic_examples/README.md

```diff
@@ -49,12 +61,14 @@ python autoencoder.py --trainer.gpus 2
 # dataparallel
 python autoencoder.py --trainer.gpus 2 --trainer.accelerator 'dp'
 ```
----
+
+______________________________________________________________________
+
 # Multi-node example
 
 This demo launches a job using 2 GPUs on 2 different nodes (4 GPUs total).
 To run this demo do the following:
 
 1. Log into the jumphost node of your SLURM-managed cluster.
-2. Create a conda environment with Lightning and a GPU PyTorch version.
-3. Choose a script to submit
+1. Create a conda environment with Lightning and a GPU PyTorch version.
+1. Choose a script to submit
```

Comment:
what's happening here?

Reply:
MD does not require any number, so in fact, if you write everywhere 9, you'll get the indexing from 1

Reply:
that's just the strangest thing ever. 
markdown should be human readable in plain format, why would anyone write it like that and why would a formatter enforce it